pkg/controller/plan/kubevirt.go

```diff
@@ -686,6 +688,20 @@ func (r *KubeVirt) createPodToBindPVCs(vm *plan.VMStatus, pvcNames []string) err
 	return nil
 }
 
+// Sets KVM requirement to the pod and container.
+func (r *KubeVirt) setKvmOnPodSpec(podSpec *core.PodSpec) {
+	nodeSelector := make(map[string]string)
+	reqLimit := make(map[core.ResourceName]resource.Quantity)
+	if *r.Plan.Provider.Source.Spec.Type == v1beta1.VSphere && !Settings.VirtV2vDontRequestKVM {
+		nodeSelector["kubevirt.io/schedulable"] = "true"
+		// Ensure that the pod is deployed on a node where /dev/kvm is present.
+		reqLimit["devices.kubevirt.io/kvm"] = resource.MustParse("1")
```

Comment:
Why don't you modify the `Limits` directly? That way you would add instead of overwrite the requirements.

Reply:
same answer as above

Reply:
btw, maybe that's the reason for setting both `devices.kubevirt.io/kvm` and `kubevirt.io/schedulable`? why is it a "limit" rather than a "request"?

Reply:
^same as above